develop.html.mako

```diff
@@ -93,7 +93,8 @@ It is a frequent occurence that pull requests submitted either don't describe
 what problem they intend to solve, or they solve the problem in an inappropriate
 or sub-optimal way.   Having a bug or feature issue opened ahead of time
 will allow us to to understand the use case and discuss the best approach
-before the contributor makes the effort to write the code and tests.</li>
+before the contributor makes the effort to write the code and tests.  The Github
```

Comment:
"The issue number used by Github is also linked in the changelog and is used to generate a changelog file entry for merge."